### DISCUSSION TITLE
我想使用event.comment.strip()获得加群时输入的答案，报错提示却是comment是个空值

### DISCUSSION THREAD

**Discussion by Ah-Q** [nonebot/discussions#452] [问答] [answered]
`无法触发的群`
![image](https://github.com/nonebot/discussions/assets/80017370/ae60453c-a352-40c8-9bf4-e4cd6f97cbe2)
`协议端`
![image](https://github.com/nonebot/discussions/assets/80017370/c4dbbb57-f5bb-4515-bdd0-45418ab7cbcb)

`入群问题`
![image](https://github.com/nonebot/discussions/assets/80017370/24e3f95a-f67d-4d8e-9eb4-28563cbe2077)

**Comment by Ah-Q** [comment]
离谱的是换个群就能获得消息
![image](https://github.com/nonebot/discussions/assets/80017370/9a46e751-771d-4e76-8df5-ac92d899998f)

不同的是协议端多了一大堆东西
![image](https://github.com/nonebot/discussions/assets/80017370/526b6ed1-3419-43bf-8830-bd54f42f563a)

**Comment by yanyongyu** [comment, marked as answer]
这个问题你应该去协议端提出

**Reply by Ah-Q** [reply]
好的 佬

**Reply by yanyongyu** [reply]
入群申请的comment是可能为None的，比如邀请加群就不会有回答问题
